### PR TITLE
Add new repo pusher channel, when new repo created

### PR DESCRIPTION
**Pull request by HelenaEksler**
#86

### CODE REVIEW

**Comment by HelenaEksler** [comment]
shoov3

File: client/app/scripts/controllers/builds.js [hunk 1]
```diff
@@ -16,22 +16,77 @@ angular.module('clientApp')
...
-            val[0].new = true;
-            $scope.builds.unshift(val[0]);
-            $timeout(function(){
-              $scope.builds[0].new = false
-            }, 1000);
+
+      // Listen for new repository was created event.
+      channel.bind('new_repo', function(data) {
+        // Add new chanel.
+        channelManager.addChannel(data.nid);
```
Comment (amitaibu) [inline client/app/scripts/controllers/builds.js]: nid -> id

**Comment by amitaibu** [comment]
needs a re-roll, and maybe some fixes based on the fixes I've done for the previous Pusher PR

**Comment by HelenaEksler** [comment]
When trying to re-create PR based on master - see all commits, that are already merged.
selection_379

**Comment by amitaibu** [comment]
I'm pretty sure I didn't commit it to `master` - can you see this code there?

**Comment by HelenaEksler** [comment]
I mean all pusher integratin is already in master, and this branch was checked out from `pusher` branch. Anyway, I've re-rolled it and created new PR based on master branch. #89

**Pull request closed without merging**